src/main/java/uk/ac/ebi/eva/pipeline/io/writers/VepAnnotationFileWriter.java

```diff
@@ -68,7 +75,9 @@ public void update(ExecutionContext executionContext) throws ItemStreamException
 
     @Override
     public void close() throws ItemStreamException {
-        vepProcess.close();
+        if (vepProcessOpened) {
```

Comment:
yes it does, because it has to handle the error output when it closes the process (among other little things